src/popup/pages/AccountDetails.vue

```diff
@@ -106,45 +108,59 @@ export default defineComponent({
     const clientHeight = ref<number>(0);
     const initialClientHeight = ref<number>(EXTENSION_HEIGHT);
 
+    // TODO flag for Ui Test
+    const isMultisig = true;
+
     const { resetFilter } = useTransactionAndTokenFilter();
 
     const appInnerElem = computed<HTMLElement | null | undefined>(
       () => accountDetailsElem.value?.parentElement,
     );
-    const isConnected = computed(() => root.$store.getters.isConnected);
-    const account = computed(() => root.$store.getters.account);
-    const activeIdx = computed(() => root.$store.state.accounts.activeIdx);
+    const activeIdx = useState('accounts', 'activeIdx');
+    const isConnected = useGetter('isConnected');
+    const account = useGetter<IAccount>('account');
+    const modalOpen = useDispatch('modals/open');
+
     const simplexLink = computed(() => buildSimplexLink(account.value.address));
     const showNamesNavigation = computed(() => !!root.$route?.meta?.showNamesNavigation);
     const routeName = computed(() => root.$route.name);
 
-    const actions = computed((): ButtonAction[] => [
-      {
-        text: root.$t('pages.token-details.receive'),
-        onClick: () => root.$store.dispatch('modals/open', {
-          name: MODAL_TRANSFER_RECEIVE,
-        }),
-        icon: ArrowReceiveIcon,
-      },
-      {
-        text: root.$t('pages.token-details.send'),
-        onClick: () => root.$store.dispatch('modals/open', {
-          name: MODAL_TRANSFER_SEND,
-        }),
-        icon: ArrowSendIcon,
-        disabled: !isConnected.value,
-      },
-      {
-        text: root.$t('pages.token-details.buy'),
-        onClick: () => window.open(simplexLink.value, '_blank'),
-        icon: CreditCardIcon,
-      },
-      {
-        text: root.$t('pages.token-details.swap'),
-        onClick: () => window.open(DEX_URL, '_blank'),
-        icon: SwapIcon,
-      },
-    ]);
+    const actions = computed((): ButtonAction[] => {
+      const actionsList = [
+        {
+          text: root.$t('pages.token-details.receive'),
+          onClick: () => modalOpen({
+            name: MODAL_TRANSFER_RECEIVE,
+          }),
+          icon: ArrowReceiveIcon,
+        },
+        {
+          text: isMultisig
+            ? root.$t('pages.token-details.propose-tx')
+            : root.$t('pages.token-details.send'),
+          onClick: () => modalOpen({
+            name: MODAL_TRANSFER_SEND,
```

Comment:
To align with the changes I'm making right now please add property `isMultisig`:
```
{
  name: MODAL_TRANSFER_SEND,
  isMultisig,
}
```